app/controllers/invitations_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
 
   def create
     if invitation.save
-      SpaceInvitationMailer.space_invitation_email(invitation).deliver_later
+      SpaceInvitationMailer.space_invitation_email(invitation, space).deliver_later
```

Comment:
The `Invitation` class exposes a `space` instance method; so we should not need to pass it in as a second parameter.

Reply:
fixed in 483e144e6167707e34e169c5abf9d2f902391108